src/main/cpp/RobotContainer.cpp

```diff
@@ -44,17 +44,32 @@ RobotContainer::RobotContainer() {
 
     // TODO: put other subsystems here.
 
+    // Initialize the Gamepads
+    VOKC_CALL(InitGamepads());
+
+    // Initialize the commands
+    VOKC_CALL(InitCommands());
+
     // Configure the button bindings
     ConfigureButtonBindings();
 }
 
 void RobotContainer::ConfigureButtonBindings() {
     // Configure your button bindings here
+    driver_back_button_->OnTrue(teleop_drive_command_.get());
+    driver_a_button_->OnTrue(slow_teleop_drive_.get())
+        .OnFalse(teleop_drive_command_.get());
+    driver_b_button_->OnTrue(quick_teleop_drive_command_.get())
+        .OnFalse(teleop_drive_command_.get());
 }
 
-frc2::Command *RobotContainer::GetAutonomousCommand() {
+std::shared_ptr<frc2::Command> RobotContainer::GetAutonomousCommand() {
     // An example command will be run in autonomous
```

Comment:
delete comment about "example command"
or, well, I guess it still is an example command, because we haven't written any autos yet.
I don't want us to forget about it though

Reply:
This will be fixed in #18